.travis.yml

```diff
@@ -35,6 +35,8 @@ notifications:
   email: false
 deploy:
   provider: puppetforge
+  edge:
+    branch: ha-bug-puppet-forge
```

Comment:
this is needed to deploy to the forge until https://github.com/travis-ci/dpl/issues/487 is merged